pkg/appruntime/app_runtime.go

```diff
@@ -151,6 +154,26 @@ func (a *Application) Run(ctx context.Context, cli client.Client, respStream cha
 	if a.Spec.DocNullReturn != "" {
 		out[base.APPDocNullReturn] = a.Spec.DocNullReturn
 	}
+	if input.ConversationID != "" {
+		conversationKnowledgebaseExist := true
+		kb := &arcadiav1alpha1.KnowledgeBase{}
+		err := cli.Get(ctx, types.NamespacedName{Namespace: a.Namespace, Name: input.ConversationID}, kb)
+		if err != nil {
+			if apierrors.IsNotFound(err) {
+				conversationKnowledgebaseExist = false
+				klog.FromContext(ctx).V(3).Info("conversation knowledgebase not exist")
+			} else {
+				return output, err
+			}
+		}
+		if conversationKnowledgebaseExist {
+			if kb.Status.IsReady() {
+				out[base.ConversationKnowledgeBaseInArg] = kb
+			} else {
+				klog.FromContext(ctx).V(3).Info("conversation knowledgebase not ready")
```

Comment:
we should return the knowledgebase ready status to the user, so the user can know whether it's fine to chat with docs.

Reply:
Currently, there is no web interface to configure and display the conversation knowledgebase; we will display and configure the conversation knowledgebase in a subsequent pull request.

cc @liutianyi0801 We need to add or update our web interface here.

Reply:
we can track this at another issue